utils/constants.tsx

```diff
@@ -30,16 +30,10 @@ import logoWebhook from 'images/webhook_icon.jpg';
 
 import {t} from 'utils/i18n';
 
-import githubCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/github.css';
-
-// eslint-disable-line import/order
-import monokaiCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/monokai.css';
-
-// eslint-disable-line import/order
-import solarizedDarkCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/solarized-dark.css';
-
-// eslint-disable-line import/order
-import solarizedLightCSS from '!!file-loader?name=files/code_themes/[hash].[ext]!highlight.js/styles/solarized-light.css'; // eslint-disable-line import/order
+import {githubCSS,
+    monokaiCSS,
+    solarizedDarkCSS,
+    solarizedLightCSS} from 'utils/css_themes';
```

Comment:
Just some nitpicking, but would you mind moving the curly brackets onto their own lines to match our other code?
```suggestion
import {
    githubCSS,
    monokaiCSS,
    solarizedDarkCSS,
    solarizedLightCSS,
} from 'utils/css_themes';
```